tests/s2n_test.h

```diff
@@ -86,6 +86,7 @@
 
 #define EXPECT_FAILURE( function_call )  do { EXPECT_EQUAL( (function_call) ,  -1 ); EXPECT_NOT_EQUAL(s2n_errno, 0); EXPECT_NOT_NULL(s2n_debug_str); s2n_errno = 0; s2n_debug_str = NULL; } while(0)
 #define EXPECT_FAILURE_WITH_ERRNO( function_call, err ) do { EXPECT_EQUAL( (function_call), -1 ); EXPECT_EQUAL(s2n_errno, err); EXPECT_NOT_NULL(s2n_debug_str); s2n_errno = 0; s2n_debug_str = NULL; } while(0)
```

Comment:
Maybe use `EXPECT_FAILURE_WITH_ERRNO_NO_RESET` in here to avoid duplicating code?

Reply:
Sure.